src/main/java/ch/admin/bag/covidcertificate/service/CovidCertificateConversionService.java

```diff
@@ -31,7 +31,7 @@ public ConvertedCertificateResponseEnvelope convertFromExistingCovidCertificate(
 
         // check if uvci of origin certificate got revoked
         final String originUvci = conversionDto.getDecodedCert().getVaccinationInfo().get(0).getIdentifier();
-        if (revocationService.isAlreadyRevoked(originUvci)) {
+        if (revocationService.getRevocationDateTime(originUvci) != null) {
             throw new ConvertCertificateException(Constants.CONVERSION_UVCI_ALREADY_REVOKED, originUvci);
```

Comment:
Might be wrong on this: Shouldn't be the revocation datetime be part of the message "UVCI_ALREADY_REVOKED"?

Reply:
Sorry Armin, die Base stimmte nicht.

Reply:
*lol*